src/silhouette/top-navigation/examples.js

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
would it make sense to have only one top level category, `Silhouettes`, and all the different layout could go under that?

Reply:
can you elaborate? So to have only one folder `silhouettes` with all files gathered in there?